spec/repository_finder_spec.rb

```diff
@@ -77,12 +77,14 @@
     end
 
     context "when git doesn't exist" do
-      before(:each) { @old_path = ENV.delete("PATH").to_s.split(File::PATH_SEPARATOR) }
-      after(:each)  { ENV["PATH"] = @old_path.join(File::PATH_SEPARATOR) }
+      before(:each) do
+        @old_path = ENV["PATH"]
+        ENV["PATH"] = ""
+      end
+      after(:each)  { ENV["PATH"] = @old_path }
 
       it "fails with a nice error message" do
         allow(subject).to receive(:git_remote_url).and_call_original
-        expect(subject.send(:git_exe_path)).to eql(nil)
```

Comment:
I think we still need to be able to run git? If we're relying on Jekyll::Utils::Exec, then we'll definitely need to make sure that the exe can be found, right?

Reply:
Relying on `Jekyll::Utils::Exec` is like running `system "git"` in a `begin..rescue` block..
If the utility method doesn't find the executable, an `Errno::ENOENT` is raised which is rescued to output a warning to the terminal..

Reply:
Ok, so we can remove `git_exe_path`?

Reply:
Yep. In fact, this PR has already removed it..